rabbitMQ/ReceivingMessage.js

```diff
@@ -0,0 +1,37 @@
+import amqp from 'amqplib';
+import addWebsite from '../controllers/website.js';
+import backupSite from '../Server.js';
+import logger from '../logger.js';
+
+export const handleData = async (data) => {
+  const functionName = data.extraParam;
```

Comment:
please, don't move the "functionName"
instead, you can send an "action"
its more clear. the name of the action and the function shouldn't be the same.